spec/controllers/api_v1_slides_controller_spec.rb

```diff
@@ -4,7 +4,7 @@
   before(:each) { request.env["HTTP_ACCEPT"] = 'application/json' }
   render_views
   let(:first_user) { create(:first_user) }
-  list_json_keys = ['id', 'user_id', 'name', 'description', 'category_id', 'key', 'extension', 'num_of_pages', 'created_at', 'category_name', 'username', 'thumbnail_url', 'transcript_url', 'tags']
+  list_json_keys = ['id', 'user_id', 'name', 'description', 'category_id', 'object_key', 'extension', 'num_of_pages', 'created_at', 'category_name', 'username', 'thumbnail_url', 'transcript_url', 'tags']
```

Comment:
__[rubocop]__
* Style/WordArray: Use `%w` or `%W` for an array of words. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-w)
* Metrics/LineLength: Line is too long. [164/160] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->